### PR TITLE
Add PHP 8.2 support

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

File: src/Adapter/Http.php [hunk 1]
```diff
@@ -479,7 +479,7 @@ protected function _basicAuth($header)
 
         // Decode the Authorization header
         $auth = substr($header, strlen('Basic '));
-        if ($auth === false) {
+        if ($auth === '') {
```
Comment (fezfez) [inline src/Adapter/Http.php:482]: this change was highlight by psalm

see change log for substr

```

8.0.0 | The function returns an empty string where it previously returned false.

```

Comment (fezfez) [reply src/Adapter/Http.php:482]: @Ocramius what do you think ?

Comment (Ocramius) [reply src/Adapter/Http.php:482]: Was a test failing here?

Comment (Ocramius) [reply src/Adapter/Http.php:482]: Yeah, we should make sure this path is hit with a test 🤔

**Comment by Ocramius** [comment]
@fezfez this targets `3.0.x` - that's the next major.

We need a `2.x` release, if you want that to get out soon-ish :D

**Comment by fezfez** [comment]
closing in favor of https://github.com/laminas/laminas-authentication/pull/45

**Comment by Ocramius** [comment]
Part of #46

**Pull request closed without merging**